lib/improver/wind_direction.py

```diff
@@ -80,16 +80,50 @@ class WindDirection(object):
     Step 6 still needs more development so it is only included in the code
     as a placeholder.
 
+    Keyword Args:
+        backup_method (str):
+            Backup method to use if the complex numbers approach has low
+            confidence.
+            "first realization" (default) uses the value of realization zero.
+            "neighbourhood" recalculates using the complex numbers approach
+            with additional realizations extracted from neighbouring grid
+            points from all available realizations.
+
     """
 
-    def __init__(self):
+    def __init__(self, backup_method='first realization'):
```

Comment:
If we're making this an optional argument, we'll need to update the CLI to accommodate this option.

Reply:
The purpose of the optional argument is to allow the plugin to call itself with a different mode. The second PR makes use of this for the neighbourhood mode - when the backup method is triggered, it applies the neighbourhood then recursively calls the plugin in the first-realization mode to avoid triggering the recursion again.